docs/network-flow-visibility.md

```diff
@@ -199,22 +199,43 @@ exploration.
 
 ### Deployment Steps
 
-To create all the necessary resources in the `elk-flow-collector` Namespace
-and get everything up-and-running, run:
+First download the necessary resources from the Antrea repository. You can either
+clone the entire repo or download the particular folder using the subversion (svn)
+utility. If the deployed version of Antrea has a release `<TAG>` (e.g. `v0.10.0`),
+then you can use the following command:
 
 ```shell
+git clone --depth 1 --branch <TAG> https://github.com/vmware-tanzu/antrea.git
```

Comment:
maybe `git clone --depth 1 --branch <TAG> https://github.com/vmware-tanzu/antrea.git && cd antrea/build/yamls` so that the following commands work the same as for subversion

Reply:
Good idea. Changed it.